guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -35,43 +35,52 @@ The process for configuring Fastly includes:
+{{ site.data.var.ece }} projects do not require a dedicated Fastly account or account owner. Instead, each project environment has its own set of Fastly credentials (API key and service ID) that you use to configure and manage Fastly services from the Magento Admin UI. You also use these credentials to submit requests to the Fastly API service. 
 
-For details, review your Fastly accounts and [documentation](https://docs.fastly.com/)
-to remove the domains. This may include removing and updating CNAME records and
-more.
+To get these credentials, Magento needs to add your project environments to the Fastly service account for {{ site.data.var.ece }} and add the Fastly account credentials to your project environment configurations.
 
-## Get your Fastly credentials {#cloud-fastly-creds}
+#### To add your project environments to the Fastly service account:
 
-To get Fastly credentials, open a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html).
-You must provide your fully-qualified domain name.
+Submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to add your environments to the Fastly service account for {{ site.data.var.ece }}:
 
-We'll provide you with the following credentials for your Staging and Production
-services:
+-  For Pro projects, include the origin URLs for your Production and Staging sites.
 
-*	Fastly Service ID
-*	Fastly API token
+-  For Starter projects, include the origin URLs for your Master and Staging sites.
 
-You can also locate these credentials in your Staging and Production systems
-in `/mnt/shared/fastly_tokens.txt`. You can SSH into the servers to verify the
-file in that location. If you do not locate this file, please enter a ticket
-for [Support]({{ page.baseurl }}/cloud/trouble/trouble.html) asking to have the
-file added. We can help provide this credentials file.
+If the domains included in your Cloud project are associated with an existing Fastly service account, see [Multiple Fastly accounts and assigned domains](#domain).
 
-{:.bs-callout .bs-callout-warning}
-Make note of which environment each set of credentials is used for. If you use
-the wrong credentials in an environment, you'll encounter issues with Fastly.
+
+####  To view your Fastly credentials:
+
+When we provision your project environments for Fastly services, we add the Fastly credentials to your project environment configuration. You can get these credentials using any of the following methods:
+
+-  After you [enable the Fastly module](#cloud-fastly-config), you can view the credentials in the Fastly configuration section of the Admin UI.
+
+   ![Fastly credentials Admin UI]({{ site.baseurl }}/common/images/cloud/cloud-fastly-credentials-admin-ui.png){:width="650px"}
+
+-  You can get credentials from the following environment variables that are visible in the [Environment configuration]({{ page.baseurl }}/cloud/project/projects.html#environment-configuration-variables) section of the Project Web UI.
```

Comment:
This list seems wordy. Is it possible to distill the information further and still provide the examples? 

- Admin UI Configuration section
- Project Web UI environment variable list
- Magento Cloud CLI variable list
- IaaS mounted shared directory